src/public_key.rs

```diff
@@ -166,6 +166,43 @@ impl From<multisig::PublicKey> for PublicKeyRepr {
     }
 }
 
+#[cfg(feature = "sqlx-postgres")]
+mod sqlx_postgres {
+    use super::*;
+    use sqlx::{
+        decode::Decode,
+        encode::{Encode, IsNull},
+        error::BoxDynError,
+        postgres::{PgArgumentBuffer, PgTypeInfo, PgValueRef, Postgres},
+        types::Type,
+    };
+
+    impl Type<Postgres> for PublicKey {
+        fn type_info() -> PgTypeInfo {
+            PgTypeInfo::with_name("text")
+        }
+    }
+
+    impl Encode<'_, Postgres> for PublicKey {
+        fn encode_by_ref(&self, buf: &mut PgArgumentBuffer) -> IsNull {
+            let address = self.to_string();
+            Encode::<Postgres>::encode(&address, buf)
+        }
+
+        fn size_hint(&self) -> usize {
+            25
```

Comment:
Is there a constant that this could be derived from?

Reply:
Not quite a constant.. it's the length of a base58 check encoded binary which is.. dependent on the binary itself, I believe